setup.py

```diff
@@ -38,21 +38,3 @@
         ]
     }
 )
-
-
-if sys.version_info[0] < 3:
-    sys.stderr.write("""
-    Heads up! You're now running ftfy 4.x on Python 2.x. This is fine, but
-    here's an advance warning that you won't be able to upgrade ftfy
-    forever without upgrading Python.
-
-    The next major version of ftfy, version 5.0, will probably only work
-    on Python 3, making it easier to develop and ensure its consistency.
-
-    It's fine if you're happy with Python 2 and ftfy 4. Save yourself a
-    headache later and be sure to pin a version in your dependencies.
-    Instead of just asking for 'ftfy', you should ask for:
-
-        'ftfy >= 4, < 5'
-    """.strip(' '))
-
```

Comment:
5.0 is going to be released at the same time, so if you got version 4.4, you already heeded the warning.